playbooks/ansible-test-cloud-integration-base/run.yaml

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
I image we'll make this playbook generic enough to support both jobs, rather then copypasta this whole thing

Reply:
Agreed.